assets/src/application/ui/layout/dropdownMenu/DropdownMenuList.tsx

```diff
@@ -2,11 +2,13 @@ import React, { Children, cloneElement } from 'react';
 import classNames from 'classnames';
 
 import { MenuList, MenuListProps } from '@infraUI/layout/menu';
+import './style.scss';
+
+const DropdownMenuList: React.FC<MenuListProps> = ({ children, placement = 'top', ...props }) => {
+	const className = classNames('ee-dropdown-menu__list', props.className);
 
-const DropdownMenuList: React.FC<MenuListProps> = ({ children, className, placement = 'top', ...rest }) => {
-	const newClassName = classNames(' components-dropdown-menu', className);
 	return (
-		<MenuList placement={placement} {...rest} className={newClassName}>
+		<MenuList {...props} className={className} placement={placement}>
```

Comment:
putting `placement` prop in front doesn't make any difference in this specific case